scss/components/_charts.scss

```diff
@@ -0,0 +1,313 @@
+// the amount of padding at the bottom of vertical bar charts
+$chart-bottom: 30px;
+// the width of the axis for vertical bar charts
+$chart-axis-width: 120px;
+
+$tooltip-arrow-size: 8px;
+$tooltip-border-color: #999;
+
+.chart-container {
```

Comment:
Why is there no `.chart` class? I feel like `.chart-container` could actually just be `.chart`.